libs/output-mapping/src/TableLoader.php

```diff
@@ -150,8 +161,14 @@ public function uploadTables(
             $loadTableTasks[] = $loadTableTask;
         }
 
+        if ($strategy instanceof Writer\Table\Strategy\SqlWorkspaceTableStrategy && $hasDirectGrant) {
```

Comment:
tady je zbytečný if na strategy, protože $hasDirectGrant je true jen když to je SqlWorkspace